server/controllers/authController.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-unused-vars */
+import { Request, Response } from 'express';
+import { sign } from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import loginValidation from '../utils/validation/loginValidation';
```

Comment:
Add the validation in the index and here import it with the CustomError from the index